test/functional/core/job_spec.lua

```diff
@@ -420,7 +427,13 @@ describe('jobs', function()
     let g:job_opts = {'on_stdout': Callback}
     call jobstart('echo "some text"', g:job_opts)
     ]])
-    eq({'notification', '1', {'foo', 'bar', {'some text', ''}, 'stdout'}}, next_msg())
+    expect_msg_seq(
+      { {'notification', '1', {'foo', 'bar', {'some text', ''}, 'stdout'} },
+      },
+      -- Alternative sequence:
+      { {'notification', '1', {'foo', 'bar', {'some text'}, 'stdout'} },
```

Comment:
Need an extra chunk since the alternative sequence didn't see the full result in the first chunk.

```
-- Alternative sequence:
{ {'notification', '1', {'foo', 'bar', {'some text'}, 'stdout'},
  {'notification', '1', {'foo', 'bar', {'', ''}, 'stdout'} },
```